index.html

```diff
@@ -1277,6 +1277,17 @@ <h2>Dereferencing the Resource</h2>
 										<li>Update the <var>selected <a>service endpoint</a> URL</var> to
 										the result of the "Reference Resolution" algorithm.</li>
 									</ol>
+									<div class="note">
+										<p>
+											Resolving <a>service endpoints</a> — particularly ones that are DIDs — might
+											result in a <dfn>resolution cycle</dfn>, which is a set of steps that result in
+											an infinite loop. For example, a <a>service endpoint</a> might indirectly point
+											back through a sequence of resolutions to a previously dereferenced identifier.
+											<a>DID resolvers</a> recursively resolving <a>service endpoints</a> are advised
+											to detect and handle such cycles to prevent infinite loops or resolution failures.
```

Comment:
```suggestion
											A <a>DID resolver</a> recursively a resolving <a>service endpoint</a> is advised
											to detect and handle such a cycle to prevent an infinite loop or resolution failure.
```